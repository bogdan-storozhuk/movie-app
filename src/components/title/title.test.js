import React from "react";
import { render } from "@testing-library/react";

import Title from "./";

describe("Title", () => {
  it("should match snapshot", () => {
    const { asFragment } = render(<Title>Example</Title>);

    expect(asFragment(<Title>Example</Title>)).toMatchSnapshot();
  });
});
