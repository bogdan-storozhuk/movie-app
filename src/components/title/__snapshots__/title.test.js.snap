// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should match snapshot 1`] = `
<DocumentFragment>
  <h1
    class="Title"
  >
    Example
  </h1>
</DocumentFragment>
`;
